src/components/RepositoryCard.stories.jsx

```diff
@@ -0,0 +1,35 @@
+import RepositoryCard from './RepositoryCard'
+
+export default {
+  title: 'RepositoryCard',
+  component: RepositoryCard,
+}
+
+function Template(args) {
+  return (
+    <RepositoryCard
+      {...args}
+    />
+  )
+}
+
+export const Default = Template.bind({})
+
+Default.args = {
+  installed: true,
+  title: 'Plural',
+  subtitle: 'One click deploys for data scientists and developers',
+  imageUrl: '/plural-logo.png',
```

Comment:
Something that might be useful is to take a darkmode image too (and use grommet's dark theme detection to toggle between them).

Reply:
😭 IMO not needed, as it's feature crawl.